### PR TITLE
podman stop --cidfile missing --ignore

### PR DESCRIPTION
**Pull request by rhatdan**
Podman should ignore failures to find a cidfile when stoping the container if the user specified --ignore

Fixes: https://github.com/containers/podman/issues/19546

#### Does this PR introduce a user-facing change?

```release-note
Podman stop will ignore cidfile not existing when user specifies --ignore flag.
```

### CODE REVIEW

File: cmd/podman/containers/stop.go [hunk 1]
```diff
@@ -124,6 +128,10 @@ func stop(cmd *cobra.Command, args []string) error {
 		stopOptions.Filters[split[0]] = append(stopOptions.Filters[split[0]], split[1])
 	}
 
+	// If no args, then nothing to stop
```
Comment (vrothberg) [inline cmd/podman/containers/stop.go:131]: This looks unrelated to the cidfile bug, is it?

**Review by vrothberg — COMMENTED** [review]
Thanks for tackling it so quickly!

File: cmd/podman/containers/stop.go (hunk 1 above)
Comment (rhatdan) [reply cmd/podman/containers/stop.go:131]: Well if you do a --cidfile=/tmp/bogus --ignore, then args will be [].  Currently this works correctly, but I do worry about calling a function that will return right  a way, in case someone down the line changes the code to report an error when args=[].

Comment (vrothberg) [reply cmd/podman/containers/stop.go:131]: Thanks!  I should have expanded the entire function.

**Review by vrothberg — APPROVED** [review]
LGTM

File: cmd/podman/containers/stop.go [hunk 2]
```diff
@@ -124,6 +128,10 @@ func stop(cmd *cobra.Command, args []string) error {
...
+	if len(args) == 0 {
```
Comment (edsantiago) [inline cmd/podman/containers/stop.go:132]: This breaks `stop -a`

Comment (rhatdan) [reply cmd/podman/containers/stop.go:132]: yikes.

Comment (rhatdan) [reply cmd/podman/containers/stop.go]: Just going to remove it and rely on the function handling it.

**Review by vrothberg — APPROVED** [review]
/lgtm
